### DISCUSSION TITLE
Where is the MacOS version storing the files?

### DISCUSSION THREAD

**Discussion by Br4thering** [LeoKlaus/Paperparrot#138] [Q&A] [answered]
Title says it all. I want to know so that I can assure proper backup.

**Comment by LeoKlaus** [comment]
All files are at `~/Library/Group Containers/group.me.wehrfritz.paperparrot/`. The original PDFs are in the "documents" folder.
There's also a button to export all documents in the settings. This will export all documents and the database to a zip file.
The zip file is not encrypted and everything in it can be opened and read (even the database).

If you have any other questions, feel free to ask!

**Reply by Br4thering** [reply]
Thanks for the prompt reply. Unfortunately there is no documents folder under that location
stefan@StefanAir group.me.wehrfritz.paperparrot % ls -l
total 2184
drwx------  6 stefan  staff     192 Oct 14 15:26 Library
-rw-r--r--@ 1 stefan  staff  647168 Oct 14 16:24 Paperparrot.sqlite
-rw-r--r--@ 1 stefan  staff   32768 Oct 14 16:15 Paperparrot.sqlite-shm
-rw-r--r--@ 1 stefan  staff  362592 Oct 14 16:28 Paperparrot.sqlite-wal
drwxr-xr-x@ 2 stefan  staff      64 Oct 14 15:26 Paperparrot_ckAssets

I have added two test documents for which the stored location is not found.

**Reply by LeoKlaus** [reply, marked as answer]
Oh, I'm sorry. Do you have iCloud enabled or disabled? Having the database in that folder but not the documents is weird.

**Reply by Br4thering** [reply]
iCloud sync was enabled. After it has been turned of, the documents folder appeared.
